AGENTS.md

```
@@ -0,0 +1,18 @@
+# Developer Notes
+
+## Documentation
+- Markdown documents live under `docs/`
+- DocBook-based manuals are in `doc/`
+
+## Package management
+- Use `uv pip install --system -e .` to install dependencies defined in `setup.py`
+- Tool configurations live in `pyproject.toml`
+
+## Testing
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)